src/components/LeaderBoard/Leaderboard.jsx

```diff
@@ -309,7 +309,16 @@ const LeaderBoard = ({
                   </Link>
                 </td>
                 <td className="align-middle">
-                  <span title="Total time">{item.totaltime}</span>
+                  <span 
+                  title="Total time"
+                  style={{
+                    color: item.totalintangibletime_hrs > 0 
+                    ? '#007bff'
+                    : 'black',
+                    fontWeight: item.totalintangibletime_hrs > 0
+                    ? 'bold'
+                    : null
```

Comment:
It's better to use css classes instead inline styles
```suggestion
                  className={item.totalintangibletime_hrs > 0 ? 'boldClass' : 'normalClass'}
```

Reply:
Updated the code! Thanks for the suggestion!